socialhome/streams/templates/streams/vue.html

```diff
@@ -0,0 +1,15 @@
+{% extends "streams/base.html" %}
+{% load staticfiles slurp_json_context string_utils %}
+
+{% block content %}
+    {% slurp_json_context %}
```

Comment:
I think this should be after the `{{ block.super }}` in the javascripts section, before the `webpack` bundle load, right? Otherwise it is slowing the page load it the browser has to process script tags here. Minor but no reason not to have it later.